nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -124,3 +124,10 @@ harness/apisupport.harness/external/launcher-12.5-distribution.zip platform/o.n.
 # only one is part of the product:
 java/libs.javacapi/external/nb-javac-jdk-17.0.1-ga-api.jar java/libs.nbjavacapi/external/nb-javac-jdk-17.0.1-ga-api.jar
 java/libs.javacapi/external/nb-javac-jdk-17.0.1-ga.jar java/libs.nbjavacapi/external/nb-javac-jdk-17.0.1-ga.jar
+
+# OCI SDK has frequently changing dependencies, so they are better kept separate
```

Comment:
Are these overlaps OK? I am inclined to believe they are. Having clashes with gradle/maven distributions seems OK to me...

Reply:
Comment was updated with more accurate - _Maven and Gradle are self-contained distributions - ignoring overlaps._
Thanks Jardo.

Reply:
I tries to update `bcpg`, but `CachingPreventsLoadingOfModuleManifestsTest.testDontLoadManifests()` started failing. As this PR is already too big, I would solve the update separately.